python/lsst/daf/io/table/sql.py

```diff
@@ -0,0 +1,938 @@
+# -*- coding: utf-8 -*-
+"""
+Collection of query wrappers / abstractions to both facilitate data
+retrieval and to reduce dependency on DB-specific API.
+"""
+
+from __future__ import print_function, division
+from builtins import str, bytes, zip, range
+
+import numpy as np
+import sqlalchemy
+import struct
+
+from lsst.afw import table as afw_table
+from sqlalchemy import select
+from sqlalchemy import Table, Column, Numeric
+from sqlalchemy.schema import MetaData, CreateTable
+from sqlalchemy.types import (SmallInteger, Integer, BigInteger,
+                              Float, String, Boolean, Text)
+from sqlalchemy.types import to_instance, TypeEngine
+from contextlib import contextmanager
+
+
+MAX_ARRAY_SIZE = 65535
+MAX_VARCHAR_SIZE = 512
+
+
+def is_dict_like(obj):
+    return hasattr(obj, '__getitem__') and hasattr(obj, 'keys')
+
+
+def is_list_like(obj):
+    return hasattr(obj, '__iter__') and not isinstance(obj, (str, bytes))
+
+
+class DatabaseError(IOError):
+    pass
+
+
+def read_sql_table(table_name, con, schema=None, column_dtypes=None,
+                   coerce_float=True, parse_dates=None, columns=None,
+                   chunksize=None):
+    """Read SQL database table into an afw table.
+
+    Given a table name and an SQLAlchemy connectable, returns an Table.
+    This function does not support DBAPI connections.
+
+    Parameters
+    ----------
+    table_name : string
+        Name of SQL table in database
+    con : SQLAlchemy connectable (or database string URI)
+    schema : string, default None
```

Comment:
missing a colon in `default None`? on line 56 and on you use the form `default: None`.